README.md

```diff
@@ -89,16 +89,16 @@ This figure was obtained with the help of an allocation from [Advanced Research
 
 ## Dependencies
 
-* **C++ compiler with C++11 support** - Compilers that have been tested include:
-  * GCC 4.7.2 and later, 
+* **C++ compiler with C++14 support** - Compilers that have been tested include:
+  * GCC 4.9 and later, 
   * Clang 3.4 and later, 
   * Apple Clang 5.0 and later, and
-  * Intel C/C++ Compiler 15 and later.
-* **Cmake** 3.0 or later
+  * ~~Intel C/C++ Compiler 17 and later.~~
+* **Cmake** 3.1 or later
 * **Eigen** - Version 3.0 and later. http://eigen.tuxfamily.org
 * **MADNESS** - While it is possible to compile MADNESS separately, we recommend compiling MADNESS automatically as part of TiledArray. Compilation of MADNESS requires the following additional prerequisites (see the [MADNESS GitHub page](https://github.com/m-a-d-n-e-s-s/madness) for details):
   * **Pthreads**
-  * **MPI-2 or MPI-3 library** - [MPICH](http://www.mpich.org), [MVAPICH](http://mvapich.cse.ohio-state.edu), and [Intel MPI](https://software.intel.com/en-us/intel-mpi-library) have been tested. Intel MPI is recommended on x86/Infiniband hardware, but any of these libraries will do. We do not recommend OpenMPI as `MPI_THREAD_MULTIPLE` support is [broken](https://github.com/open-mpi/ompi/issues/157).
+  * **MPI-2 or MPI-3 library** - [MPICH](http://www.mpich.org), [MVAPICH](http://mvapich.cse.ohio-state.edu), and [Intel MPI](https://software.intel.com/en-us/intel-mpi-library) have been tested. Intel MPI is recommended on x86/Infiniband hardware, but any of these libraries will do. ~~We do not recommend OpenMPI as `MPI_THREAD_MULTIPLE` support is [broken](https://github.com/open-mpi/ompi/issues/157).~~
```

Comment:
Is the OpenMPI threading still an issue? I think 3.0 is fine, they enabled thread_multiple by default and even use MADNESS (which is much more fine grained than we are) for performance regression.

Reply:
It is fixed. I added `~~ ~~` for strike over this line, to point out this line is outdated. But I think we can just delete this whole line.